mlir/test/python/ir/lit.local.cfg

```diff
@@ -0,0 +1,2 @@
+if "Windows" in config.host_os:
+    config.excludes.add("auto_location.py")
```

Comment:
This actually works fine on Windows but the paths are different and I can't figure out how to massage the lit `RUN` lines to selectively do something on different on Windows vs Linux.